src/main/resources/application.properties

```diff
@@ -0,0 +1,40 @@
+############################################
+###       APPLICATION PARAMETERS         ###
+############################################
+
+spring.application.name=blacklist
+server.address=0.0.0.0
+server.port=8444
```

Comment:
8444 is already taken by authentication system. Please select an other one (check old Arrowhead systems to not be in conflict)